src/hotspot/cpu/riscv/vtableStubs_riscv.cpp

```diff
@@ -181,7 +181,7 @@ VtableStub* VtableStubs::create_itable_stub(int itable_index) {
   const Register resolved_klass_reg = x30; // resolved interface klass (REFC)
   const Register temp_reg           = x28;
   const Register temp_reg2          = x29;
-  const Register icdata_reg         = t1;
+  const Register icdata_reg         = t0;
```

Comment:
It seems to me quite risky to let `t0` hold CompiledICData. Here for this case, the `__ increment(Address(x18));` at L166 would clobber `t0` implicitly. I am wondering if we have other choices.

Reply:
Yes, I think I like that, thanks.

Reply:
Fixed, take another look.